_includes/nav.html

```diff
@@ -20,12 +20,12 @@
                     {% if page.titleDisplay %}{{ page.titleDisplay }}{% else %}{% endif %}
                  </a>
                 {% assign sorted2_pages = site.pages | sort: "order" %}
-                 <ul class="nav-item-sub">
-                    {% for page2 in sorted2_pages %}
+                 <ul class="nav-item-sub" aria-hidden="true">
+                    {%- for page2 in sorted2_pages -%}
```

Comment:
this literally blew my mind.... fuuuuu I should have included this on that other thing